Gini-iOS-SDK/GINIDocument.h

```diff
@@ -58,10 +58,10 @@ typedef NS_ENUM(NSUInteger, GiniDocumentSourceClassification) {
 @property GiniDocumentSourceClassification sourceClassification;
 /// Links to related resources, such as extractions, document, processed or layout.
 @property (readonly) GINIDocumentLinks *links;
-/// (Optional) Array containing the path of every parent
-@property (readonly) NSArray<NSString *> *parents;
-/// (Optional) Array containing the path of every partial document
-@property (readonly) NSArray<NSString *> *partialDocuments;
+/// (Optional) Array containing the path of every composite document
+@property (readonly) NSArray<NSString *> *compositeDocuments;
```

Comment:
This is a breaking change, though. I hope you are planning a major release.

Reply:
It is not. Those properties weren't available in previous releases and they can still use the old way. But yes, we will need to have a good documentation for this partial/composite thingy.